app/src/main/java/com/io/moviO/movieDetails/MovieDetailsFragment.kt

```diff
@@ -30,9 +30,17 @@ class MovieDetailsFragment : Fragment(R.layout.fragment_movie_details) {
                         binding.apply {
                             movieName.text = it.value.title
                             movieReleaseDate.text = it.value.releaseDate
-                            movieGenre.text = convertGenres(it.value.genres)
+                            if (it.value.genres?.isEmpty() == true) {
```

Comment:
hows this instead?

`movieGenre.text = it.value.genres?.run {convertGenres(this) } ?: getString(R.string.empty_list_genres)`

Reply:
Cool